hibernate-jpa/src/main/resources/META-INF/native-image/io.micronaut.sql/hibernate-jpa-graal/reflect-config.json

```diff
@@ -0,0 +1,16 @@
+[
+  {
+    "name": "org.hibernate.internal.log.ConnectionInfoLogger_$logger",
```

Comment:
This is alrady in graalvm metadata repo but not in the version we are using. Will remove later when not needed.